base/show.jl

```diff
@@ -2516,9 +2517,117 @@ function show_tuple_as_call(io::IO, name::Symbol, sig::Type;
     end
     print_within_stacktrace(io, ")", bold=true)
     show_method_params(io, tv)
+    str = String(take!(unwrapcontext(io)[1]))
+    if get(out, :limit, false)::Bool
+        sz = get(out, :displaysize, (typemax(Int), typemax(Int)))::Tuple{Int, Int}
+        str_lim = type_depth_limit(str, sz[2])
```

Comment:
Can we make this min(width,120), so that very skinny terminals still get a reasonable amount of output (particularly for copy paste)?

Reply:
If the concern is skinny terminals this should be `max(width, 120)` (not `min`)...but it seems like Jeff caught that.